plugins/@grouparoo/hubspot/src/lib/export-objects/destinationMappingOptions.ts

```diff
@@ -97,23 +101,27 @@ export const getObjectFields = (
   important?: boolean;
 }> => {
   const properties = customObject.getProperties();
+  const requiredFields = customObject.getRequiredProperties();
```

Comment:
in the future, it might be better to pass in `requiredFields` above.
```
    const required = getRequiredFields(customObject, primaryKey as string);
    const known = getObjectFields(customObject, required, primaryKey as string);
```
Then do `!requiredFields.includes(property["name"])`. That allows us to remove the knowledge of what we chose as `required` into just the `getRequiredFields` function and makes it so getObjectFields will never duplicate. For example, then you'd remove `property["name"] !== primaryKey`